src/all_types.hpp

```diff
@@ -639,10 +648,14 @@ struct AstNodeFnProto {
     AstNode *align_expr;
     // populated if the "section(S)" is present
     AstNode *section_expr;
+    // populated if the "callconv(S)" is present
+    AstNode *callconv_expr;
     Buf doc_comments;
 
     FnInline fn_inline;
-    CallingConvention cc;
+    bool is_nakedcc;
+    bool is_stdcallcc;
+    bool is_async;
```

Comment:
Feel free to delete nakedcc and stdcallcc keywords from stage1. (If you don't do it, I'll do it.) I'm happy to tell people to use the auto-fixing you added to `zig fmt` in order to upgrade code to the latest version.

I'll open a separate proposal to remove support for `async fn` in favor of `callconv(.Async)` for uniformity.

Reply:
Ditto, if you want to drop `nakedcc` and `stdcallcc` in this release cycle go for it! The less legacy cruft the better.